libs/src/utils/types.ts

```diff
@@ -22,3 +22,129 @@ export interface Logger {
    */
   debug?: (message: any, ...optionalParams: any[]) => any
 }
+
+type CID = string
+type ID = number
+type UID = string
+
+export type UserMetadata = {
+  album_count: number
+  bio: string | null
+  cover_photo: Nullable<CID>
+  creator_node_endpoint: Nullable<string>
+  current_user_followee_follow_count: number
+  does_current_user_follow: boolean
+  followee_count: number
+  follower_count: number
+  supporter_count: number
+  supporting_count: number
+  handle: string
+  handle_lc: string
+  is_deactivated: boolean
+  is_verified: boolean
+  location: Nullable<string>
+  // this should be removed
+  is_creator: boolean
+  name: string
+  playlist_count: number
+  profile_picture: Nullable<CID>
+  repost_count: number
+  track_count: number
+  cover_photo_sizes: Nullable<CID>
+  profile_picture_sizes: Nullable<CID>
+  metadata_multihash: Nullable<CID>
+  has_collectibles: boolean
+  collectiblesOrderUnset?: boolean
+
+  // Only present on the "current" account
+  does_follow_current_user?: boolean
+  track_save_count?: number
+  user_id: number
```

Comment:
Is `user_id` only present on the "current" account? Probably not right? but if so it should probably be optional